elasticsearchdashboard/ed_client_v8.go

```diff
@@ -167,15 +167,36 @@ func (h *EDClientV8) ListSpaces() ([]string, error) {
 		return nil, fmt.Errorf("failed to list dashboard spaces %s", string(body))
 	}
 
-	var spaces []map[string]interface{}
+	var spaces []Space
 	if err = json.Unmarshal(body, &spaces); err != nil {
 		return nil, err
 	}
 
-	var spacesName []string
-	for _, space := range spaces {
-		spacesName = append(spacesName, space["id"].(string))
+	return spaces, nil
+}
+
+func (h *EDClientV8) CreateSpace(space Space) error {
+	req := h.Client.R().
+		SetDoNotParseResponse(true).
+		SetHeaders(map[string]string{
```

Comment:
Can we keep this map in a constant and re-use everywhere? May be name it `jsonHeaderForKibanaAPI`
```
map[string]string{
"Content-Type": "application/json",
"kbn-xsrf":     "true",
}
```